src/it/java/org/jboss/tools/intellij/kubernetes/fixtures/mainIdeWindow/LinuxIdeMenuBarFixture.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.intellij.kubernetes.fixtures.mainIdeWindow;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.intellij.remoterobot.data.RemoteComponent;
+import com.intellij.remoterobot.fixtures.*;
+import org.jetbrains.annotations.NotNull;
+
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
+
+/**
+ * Top menu fixture for Linux
+ *
+ * @author zcervink@redhat.com
+ */
+
+@DefaultXpath(by = "LinuxIdeMenuBar type", xpath = "//div[@class='LinuxIdeMenuBar']")
```

Comment:
ok, so it wont work on mac, windows. It only works on Linux. Since we have github actions to run it, good enough for now. Is it possible to have these runnable on Mac/Windows, too, on the long run?

Reply:
1."Since we have github actions to run it, good enough for now." : Do we already have a github action to run it? I thought we needed to create it.
2. "Is it possible to have these runnable on Mac/Windows, too, on the long run?" : It looks like "yes", but I don't know "how" yet, so I have no answer, sorry :(